Formula/opencascade.rb

```diff
@@ -0,0 +1,73 @@
+class Opencascade < Formula
+  desc "3D modeling and numerical simulation software for CAD/CAM/CAE"
+  homepage "https://dev.opencascade.org/"
+  url "https://github.com/FreeCAD/homebrew-freecad/releases/download/0/opencascade-7.2.0.tgz"
+  sha256 "f9a81ca2267635773af9fff80625735dd4f65389af986fe93a94263081b422e9"
+
+  option "with-extras", "Install documentation (~17 MB), source files (~113 MB), samples and templates"
+
+  depends_on "cmake" => :build
+  depends_on "freetype"
+  depends_on "freeimage"
+  depends_on "tbb"
+  depends_on "gl2ps"
+  depends_on "doxygen" if build.with? "extras"
+
+  def install
+    cmake_args = std_cmake_args
+    sdk = MacOS::CLT.installed? ? "" : MacOS.sdk_path
+    opencl_path = Pathname.new "#{sdk}/System/Library/Frameworks/OpenCL.framework/Versions/Current"
+    cmake_args << "-DCMAKE_PREFIX_PATH:PATH=#{HOMEBREW_PREFIX}"
+    cmake_args << "-DCMAKE_INCLUDE_PATH:PATH=#{HOMEBREW_PREFIX}/lib"
+    cmake_args << "-DCMAKE_FRAMEWORK_PATH:PATH=#{HOMEBREW_PREFIX}/Frameworks"
+    cmake_args << "-D3RDPARTY_DIR:PATH=#{HOMEBREW_PREFIX}"
```

Comment:
We should find which formula this is looking for, and use `Formula["xxx"].opt_prefix` here also